weed/shell/command_volume_fix_replication.go

```diff
@@ -60,6 +60,7 @@ func (c *commandVolumeFixReplication) Do(args []string, commandEnv *CommandEnv,
 	noDelete := volFixReplicationCommand.Bool("noDelete", false, "Do not delete over-replicated volumes, only fix under-replication")
 	retryCount := volFixReplicationCommand.Int("retry", 5, "how many times to retry")
 	volumesPerStep := volFixReplicationCommand.Int("volumesPerStep", 0, "how many volumes to fix in one cycle")
+	replication := volFixReplicationCommand.String("replication", "", "Customize the number of replicas instead of following the setting when the volume was created, specify this parameter")
```

Comment:
After a different replication setting, the volume would be considered wrongly replicated.

when this is needed?

Reply:
got it，I should have used "volume.configure.replication", then "volume.fix.replication".